src/test/java/com/paloaltonetworks/osc/api/AbstractPANApiIntegrationTest.java

```diff
@@ -0,0 +1,79 @@
+package com.paloaltonetworks.osc.api;
+
+import static com.paloaltonetworks.utils.SSLContextFactory.getSSLContext;
+
+import javax.net.ssl.HostnameVerifier;
+import javax.net.ssl.SSLSession;
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.ClientBuilder;
+
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.osc.sdk.manager.element.ApplianceManagerConnectorElement;
+import org.osc.sdk.manager.element.ManagerDeviceElement;
+import org.osc.sdk.manager.element.VirtualSystemElement;
+
+import com.paloaltonetworks.panorama.api.methods.PanoramaApiClient;
+
+public abstract class AbstractPANApiIntegrationTest {
+
+    protected static final String EXISTING_POLICY_TAG = "EXISTING_POLICY_TAG";
+    protected static final String EXISTING_POLICY_TAG_OTHER = "EXISTING_POLICY_TAG_OTHER";
+    protected static final long VS_ID = 32123L;
+
+    private static final long DEVGPS_SLEEP_MS = 10;
+    private static final int DEVGPS_TIMEOUT_TRIES = 10;
+    private static final String PANORAMA_IP = "10.3.240.15";

```

Comment:
can this be a public shared field so folks dont have to update the panorama IP in multiple places for their integration tests?

PanoramaApiClientIntegrationTest has the same fields.